configs/src/networks/getters.ts

```diff
@@ -16,9 +16,13 @@ export function getNetwork(network: SupportedNetworks): NetworkConfig | null {
 }
 
 export function getNetworkByChainId(chainId: number): NetworkConfig | null {
-  return (
-    Object.values(networks).find(network => network.chainId === chainId) || null
-  );
+  return chainId === 0
+    ? networks[SupportedNetworks.DEV_SEPOLIA]
```

Comment:
Do we really need the change here ? Just putting `DEV_SEPOLIA` below should never return it - i.e they always will get SEPOLIA only with this function which in my opinion is better than this change as we don't have to maintain DEV_SEPOLIA handling differently. 

May I know what made you go with this way ?

Reply:
I totally agree we should never need to get this test or use `getNetworkByChainId` at all. 

However, there are tests checking this getter works, that's the reason why I added this change and also updated [the tests](https://github.com/aragon/osx-commons/pull/96/files#diff-673696fcfc6a39a8ebf3e7b616768f7d6b86443fa13ba662beda2bed09283744).

So even when we might never need to get the DEV_SEPOLIA info, there will be a way to do so. 

**We could also modify the test to avoid checking DEV_SEPOLIA, but it seemed awkward for me to go that way, wdyt?**

Reply:
We agreed to let it be as it is because it will allow us to get the DEV_SEPOLIA information if needed.